streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapterTest.java

```diff
@@ -93,7 +114,7 @@ public void shouldOverwriteAllOptionsMethods() throws Exception {
         for (final Method method : Options.class.getMethods()) {
             if (!ignoreMethods.contains(method.getName())) {
                 RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.class
-                    .getDeclaredMethod(method.getName(), method.getParameterTypes());
+                        .getDeclaredMethod(method.getName(), method.getParameterTypes());
```

Comment:
Could you please revert this change since it is unrelated to the PR?

Reply:
@cadonna 
done, no idea how it came in...